src/home.js

```diff
@@ -1,168 +1,96 @@
-      <div class="card-img-overlay">
-        <h5 class="card-title">Card title</h5>
-        <p class="card-text">This is a wider card with supporting text below as a natural lead-in to additional content. This content is a little bit longer.</p>
-        <p class="card-text">Last updated 3 mins ago</p>
-      </div>
-    </div>
-  </div>
-</div>
-
-</div>
-
-
-`;
-
-const renderHome = () => {
-  content.insertAdjacentHTML('beforeend', slider);
-  content.insertAdjacentHTML('beforeend', aboutInfo);
-};
-
-export default renderHome;
\ No newline at end of file
+const landingPageImg = 'https://images.unsplash.com/photo-1504674900247-0877df9cc836?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=750&q=80';
+
+// Initial load for Home tab
+const initialLoad = () => {
+  let homeMainContainer = document.createElement('div');
+  homeMainContainer.setAttribute('id', 'home-main-container');
+
+  // elements that belong to mainContainer
+  let landingImg = document.createElement('img');
+  landingImg.setAttribute('src', landingPageImg);
+  landingImg.setAttribute('id', 'landing-img');
+  landingImg.setAttribute('alt', 'landing-page');
+
+  let phraseBox = document.createElement('div');
+  phraseBox.setAttribute('id', 'phrase-box');
+
+  let mainPhrase = document.createElement('p');
+  mainPhrase.setAttribute('id', 'main-phrase');
+  mainPhrase.innerHTML = 'WELCOME TO TASTY BITE';
+
+  // Appending elements to the Main
+  phraseBox.appendChild(mainPhrase);
+  homeMainContainer.appendChild(landingImg);
+  homeMainContainer.appendChild(phraseBox);
+
+  return homeMainContainer;
+}
+
+// Function for the initial setup that's repeated again and again
+const repeatedSetup = () => {
+  /**
+   * 1. Making all elements first that are in 'index.html'
+   */
+  
+  // container, header, main
+  let container = document.createElement('div');
+  container.setAttribute('id', 'container');
+  
+  let header = document.createElement('header');
+  let main = document.createElement('main');
+  
+  // Container - Direct elements
+  let headerContainer = document.createElement('div');
+  headerContainer.setAttribute('id', 'header-container');
+
+
+  // headerContainer elements
+  let restaurantTitle = document.createElement('h1');
+  restaurantTitle.setAttribute('id', 'restaurant-title');
+  restaurantTitle.innerHTML = '<span id="kor">TASTY</span>BITE';
+
+
+  let navBar = document.createElement('ul');
+  navBar.setAttribute('id', 'nav-bar');
+
+  // navBar elements
+  let navBarlis = [];
+  for(let i = 0; i < 4; i++) {
+    let li = document.createElement('li');  
+
+    switch(i) {
+      case 0:
+        li.innerHTML = 'Home';
+        break;
+      case 1:
+        li.innerHTML = 'About';
+        break;
+      case 2:
+        li.innerHTML = 'Menu';
+        break;
+      case 3:
+        li.innerHTML = 'Contact';
+        break;
+    }
+
+    navBarlis.push(li);   
+  }
```

Comment:
Let separate the DOM from application logic, you can create another file for login or create another module/function for it.

![image](https://user-images.githubusercontent.com/31897434/73420341-70e18b80-4322-11ea-840a-fa14e04b592b.png)

https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59623090/